tests/daemon-local.cfg

```diff
@@ -0,0 +1,34 @@
+# Config file for testing from a workstation (node) against a local kafka broker.
```

Comment:
"daemon-local.cfg" is a bad name for this, and where do we use it that it needs to live in this directory?